packages/garbo/src/fights.ts

```diff
@@ -215,6 +218,7 @@ const secondChainMacro = () =>
   Macro.if_(
     globalOptions.target,
     Macro.externalIf(isStrongScaler(globalOptions.target), Macro.delevel())
+      .if_("hppercentbelow 30", Macro.tryItem($item`New Age healing crystal`))
```

Comment:
Is there a world where the mime is outhitting healing crystals? should we consider funkslinging?

Reply:
The logs I've seen (including my own!) have universally been sub-500 damage in a fight near the end of the chain, with most being in the 100-150 damage range. We could funksling, but I'm worried it will be overkill. These are cheap though. Your call.